### ISSUE TITLE
dosen't support Laravel 7.6

### ISSUE THREAD

**Issue by TheAbhishekIN** [brozot/Laravel-FCM#188]
I trying to install Laravel-FCM on laravel 7.6 and it shows the errors.
here attached ss https://prnt.sc/syoh3s

**Comment by alejandrozorita** [comment]
Yep...you can see this issue. There are a one solution with other fork, for now, is the only solution.

https://github.com/brozot/Laravel-FCM/issues/175

**Comment by alejandrozorita** [comment]
Waiting for merge.
https://github.com/brozot/Laravel-FCM/pull/174
